packages/gateway/src/features/user/root-resolvers.test.ts

```diff
@@ -504,10 +504,10 @@ describe('User root resolvers', () => {
         }
       ])
     })
-    it('should return error if invalid data received from user-mgnt endpoint', async () => {
+    it('returns empty results if invalid data received from user-mgnt endpoint', () => {
       fetch.mockResponseOnce(JSON.stringify({}))
 
-      expect(
+      return expect(
```

Comment:
I noticed these two tests failing quietly as the `expect` call wasn't returned back to Jest. Updated the expected values to be what the code returns